### ISSUE TITLE
loss很高

### ISSUE THREAD

**Issue by eain3314** [qqwweee/keras-yolo3#470]
![06RR$_VV%1I{ 4L9BIBY023](https://user-images.githubusercontent.com/48511773/58314949-26c71280-7e43-11e9-8f75-76ae29904d2b.png)
测试的准确度也不行

**Comment by eain3314** [comment]
50左右

**Comment by lyqlola** [comment]
遇到同样的问题，train loss和val loss都在50以上

**Comment by eain3314** [comment]
解决了吗

**Comment by lyqlola** [comment]
并没有，加个qq聊一下？你是train from scratch，还是用了预训练权重的？

**Comment by lyqlola** [comment]
我的邮箱lyq12121@gmail.com

**Comment by eain3314** [comment]
864770486

**Comment by QLMX** [comment]
可以试试不加载预训练模型.
将train.py  :https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/train.py#L29
改成下面这种试试看 
 if is_tiny_version:
        model = create_tiny_model(input_shape, anchors, num_classes,
            load_pretrained=False, freeze_body=2, weights_path='model_data/tiny_yolo_weights.h5')
    else:
        model = create_model(input_shape, anchors, num_classes,
           load_pretrained=False,  freeze_body=2, weights_path='model_data/yolo_weights.h5') # make sure you know what you freeze

**Comment by lyqlola** [comment]
> 可以试试不加载预训练模型.
> 将train.py :
> 
> https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/train.py#L29
> 
> 
> 改成下面这种试试看
> if is_tiny_version:
> model = create_tiny_model(input_shape, anchors, num_classes,
> load_pretrained=False, freeze_body=2, weights_path='model_data/tiny_yolo_weights.h5')
> else:
> model = create_model(input_shape, anchors, num_classes,
> load_pretrained=False, freeze_body=2, weights_path='model_data/yolo_weights.h5') # make sure you know what you freeze

你试过么？效果怎么样？

**Comment by QLMX** [comment]
我训练我自己模型还可以

**Comment by eain3314** [comment]
你训练的什么数据集

**Comment by casstio** [comment]
同样很大，你可以试试不加载预训练模型，还有就是有个问题请问你是保存下来这些临时文件的？

**Comment by tslgithub** [comment]
me  too，只识别一个类别，val_loss都很高，切是完全自己训练，没有加载预训练权重

**Comment by jetsmith** [comment]
我在coco上训练tiny-yolov3，train from scratch, 目前train loss和val loss都是50多，不知道往下训会遇到同样的问题吗？单卡1080训，batch-size=64, 先用3e-4 做了一个epoch的warmup，然后用lr 1e-3训练了12epoch，train loss从2000多降到53, val loss 目前也是50多，loss下降的比较慢．你们在coco上从头训能达到多少mAP?
